backend/src/validators/common_schema/index.js

```diff
@@ -8,7 +8,7 @@ const MaxStringLength = 255;
  * Collection of commonly used field validations.
  */
 const Schema = {
-  Money: S.oneOf([S.number().multipleOf(0.01), S.const("")]),
+  Money: S.string().pattern("^(\\$)?-?\\d{1,3}(,\\d{3})*(\\.\\d{2})?$"),
```

Comment:
This makes more sense than the multipleOf, but I get how it would work with a money type field that's been cast to float